sdk/appconfiguration/app-configuration/test/public/secretReference.spec.ts

```diff
@@ -163,4 +163,25 @@ describe("AppConfigurationClient - SecretReference", () => {
       await client.deleteConfigurationSetting({ key: secondSetting.key });
     });
   });
+
+  describe("serializeAsConfigurationSettingParam", () => {
+    [`[]`, "Hello World"].forEach((value) => {
+      it(`Unexpected value ${value} as secret reference value`, async () => {
+        const setting: ConfigurationSetting<SecretReferenceValue> = {
+          contentType: secretReferenceContentType,
+          key: recorder.getUniqueName("name-1"),
+          isReadOnly: false,
+          value: { secretId: "id" }
+        };
+        setting.value = value as any;
+        await client.addConfigurationSetting(setting);
+        assert.equal(
+          (await client.getConfigurationSetting({ key: setting.key })).value,
+          value as any,
```

Comment:
Similar comment here - no need to cast to 'any', I'm guessing it's a string.

Reply:
Updated